playground.js

```diff
@@ -225,7 +225,10 @@ const startRepl = (sdk, rawMode, src) =>
           const replInstance = repl.start('> ');
 
           // Attach history
-          replInstance.setupHistory('./.repl_history', () => null);
+          // Node versions prior to 10.11 don't support setupHistory
+          if (replInstance.setupHistory !== undefined) {
```

Comment:
Comment: Most likely this will work just fine, but maybe `if (replInstance.setupHistory) { ... }` would be better? In the end, you want to check if something exist, not if something does not not exist :) Also, checking for the positive case will work correctly even if for some weird reason, I don't know why, `setupHistory` would be `null`.